app/javascript/packs/constants/index.js

```diff
@@ -1,5 +1,13 @@
 export const LOAD_INITIAL_STATE = 'LOAD_INITIAL_STATE';
 
+// Answer ActionTypes
+export const FETCH_ANSWER = 'FETCH_ANSWER';
+export const ADD_ANSWER = 'ADD_ANSWER';
+export const CREATE_ANSWER = 'CREATE_ANSWER';
+export const UPDATE_ANSWER = 'UPDATE_ANSWER';
+export const REMOVE_ANSWER = 'REMOVE_ANSWER';
+export const SAVE_ANSWER = 'SAVE_ANSWER';
```

Comment:
remove this and move all logic for save answer to update answer for naming conventions